aggregator/options.go

```diff
@@ -122,14 +128,15 @@ func NewOptions() Options {
 		metricPrefix:              defaultMetricPrefix,
 		counterPrefix:             defaultCounterPrefix,
 		timerPrefix:               defaultTimerPrefix,
-		timerSumSuffix:            defaultTimerSumSuffix,
-		timerSumSqSuffix:          defaultTimerSumSqSuffix,
-		timerMeanSuffix:           defaultTimerMeanSuffix,
-		timerLowerSuffix:          defaultTimerLowerSuffix,
-		timerUpperSuffix:          defaultTimerUpperSuffix,
-		timerCountSuffix:          defaultTimerCountSuffix,
-		timerStdevSuffix:          defaultTimerStdevSuffix,
-		timerMedianSuffix:         defaultTimerMedianSuffix,
+		lastSuffix:                defaultAggregationLastSuffix,
```

Comment:
Right now there is no sum/last suffix for counters/gauges, how do we plan to handle the backward compatibility here?